pkg/iosrc/error.go

```diff
@@ -0,0 +1,25 @@
+package iosrc
+
+import (
+	"errors"
+	"fmt"
+)
+
```

Comment:
Instead of defining a new error type & utilities, could you use the `zqe` package? 

So `NewPathError(uri, err)` would become `zqe.E(zqe.NotFound, uri.String())`?

Reply:
sure.